src/lib/components/Feedback/index.js

```diff
@@ -6,18 +6,51 @@ import {BaseElement} from '../BaseElement';
 import {trackEvent} from '../../analytics';
 import './_styles.scss';
 
+/** @types {WebFeedback.Question[]} */
+const questions = [
+  {text: 'Was this page helpful?', label: 'helpful', name: 'Helpfulness'},
+  {
+    text: 'Did this page help you complete your goal(s)?',
+    label: 'effective',
+    name: 'Effectiveness',
+  },
+  {
+    text: 'Did this page have the information you needed?',
+    label: 'complete',
+    name: 'Completeness',
+  },
+  {
+    text: "Was this page's information accurate?",
+    label: 'accuracy',
+    name: 'Accuracy',
+  },
+  {text: 'Was this page easy to read?', label: 'readable', name: 'Readability'},
+];
+
+/** @types {WebFeedback.Question[]} */
+const conditionalQuestions = [
+  {text: 'Does this API meet your needs?', label: 'api', name: 'API'},
+];
+
 /**
  * Element that handles feedback on posts to submit to Analytics.
  *
  * @extends {BaseElement}
  * @final
  */
 class Feedback extends BaseElement {
+  static get properties() {
+    return {
+      conditionals: {type: String},
+    };
+  }
+
   constructor() {
     super();
     this.robotName = 'is-it-just-me-or-was-this-form-filled-out-by-a-robot';
     this.submitted = false;
     this.submit = this.submit.bind(this);
+    this.conditionals = '';
```

Comment:
If you use the converter suggestion above then you can set this to `this.conditionals = []`